test/source/browser/controllable.ts

```diff
@@ -420,8 +420,11 @@ abstract class ControllableBase {
 
   public checkIfImageIsDisplayedCorrectly = async (selector: string) => {
     const isImageDisplayedCorrectly = await this.target.evaluate(selector => {
+      const pgpBlock = document.querySelector('#pgp_block') as HTMLDivElement;
       const img = document.querySelector(selector) as HTMLImageElement;
-      return img.naturalWidth !== 0 && img.naturalHeight !== 0;
+      const imgWidth = img.offsetHeight;
+      const pgpBlockWidth = pgpBlock.offsetHeight;
```

Comment:
It should be `img.offsetWidth` and `pgpBlock.offsetWidth`, not `offsetHeight`

Reply:
Sorry, my bad. I unconsciously use `offsetHeight` where I need to check `offsetWidth`.